### PR TITLE
Kinetic Weaponry

### PR DESCRIPTION
**Pull request by kestrel1110**
new plugin

if im doing this wrong let me know

also does the autoupdate feature work yet?
i forgor

### CODE REVIEW

**Comment by kestrel1110** [comment]
sorry my fork is so messy :/

File: manifests/KineticWeaponry.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Kinetic Weaponry
+authors: Bereskatuket (Kestrel1110)
+homepage: https://github.com/kestrel1110/KineticWeaponry
+license: GPL-3.0-or-later
+version: '1.0'
+shortDescription: Adds current-day weaponry to Endless-Sky
+description: A plugin adding modern realistic weapons to Endless-Sky. They are tier 1 human weapons, already balanced to work as anti-capital ship weapons.
+url: https://github.com/kestrel1110/KineticWeaponry/archive/1.0.zip
+iconUrl: https://github.com/kestrel1110/KineticWeaponry/raw/1.0/icon.png
```
Comment (MCOfficer) [inline manifests/KineticWeaponry.yaml:9]: Your plugin doesn't have an icon yet, so this is a 404.

**Review by MCOfficer — CHANGES_REQUESTED** [review]
Sorry, that slipped past me.

This PR is a mess because it's coming from your master branch, generally you want to create new branches from this repo's master and PR them back to here. That way you don't have to involve your master in the first place.

This time I can pick the file and commit it manually, when it's merge-ready.

File: manifests/KineticWeaponry.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Kinetic Weaponry
+authors: Bereskatuket (Kestrel1110)
+homepage: https://github.com/kestrel1110/KineticWeaponry
+license: GPL-3.0-or-later
+version: '1.0'
+shortDescription: Adds current-day weaponry to Endless-Sky
+description: A plugin adding modern realistic weapons to Endless-Sky. They are tier 1 human weapons, already balanced to work as anti-capital ship weapons.
```
Comment (MCOfficer) [inline manifests/KineticWeaponry.yaml:7]: "Endless-Sky" with a minus?

File: manifests/KineticWeaponry.yaml (hunk 1 above)
Comment (kestrel1110) [reply manifests/KineticWeaponry.yaml:9]: got it
ill get one

File: manifests/KineticWeaponry.yaml (hunk 2 above)
Comment (kestrel1110) [reply manifests/KineticWeaponry.yaml:7]: that's a dash on my keyboard
idk
you want me to change it?

File: manifests/KineticWeaponry.yaml (hunk 1 above)
Comment (kestrel1110) [reply manifests/KineticWeaponry.yaml:9]: icon is added

File: manifests/KineticWeaponry.yaml (hunk 2 above)
Comment (kestrel1110) [reply manifests/KineticWeaponry.yaml]: i just removed it

**Comment by kestrel1110** [comment]
Gonna just make a new pr, this is absurdly messy

**Pull request closed without merging**